customer-docs/schema-evolution.md

```diff
@@ -171,7 +171,7 @@ CSV is complicated as Snowflake uses ordinal position to identify column. Howeve
 
 1. Drop column on the Snowflake side.
 2. Re-define the PIPE object to remove the use of the deleted column.
-3. Move table config file from _state-directory_ to _previous-state-directory_.
+3. Move table config file from _state-directory_ to _previous-state-directory_ or remove the deleted column details from _previous-state-directory_ table config file
```

Comment:
In case of CICD, user might not be able to access the state-directory. In that case we can remove the deleted column details from prev folder